front/style/content/_movement.scss

```diff
@@ -166,14 +166,30 @@
     width: 100%;
 }
 
-#executive-officer .team {
-    justify-content: center !important;
+#biography {
+    .executive-officer {
+        .team {
+            justify-content: center !important;
```

Comment:
Why do you use !important ?

Reply:
It's an override of this property : https://github.com/EnMarche/en-marche.fr/blob/master/front/style/content/_movement.scss#L51